### DISCUSSION TITLE
Stop and remove containers on window exit

### DISCUSSION THREAD

**Discussion by g-kartik** [devcontainers/community#89] [Ideas]
We should have an optional feature for stopping and removing the dev containers on the exit of the vs code window or when user chooses to close the remote connection. It shall be helpful because the containers keep running in the background consuming memory and CPU processing.

**Comment by eljog** [comment]
Thank you for the feature request.

Since devcontainer specification and the CLI implementation are agnostic of the editor tools, I think this feature may not come under the scope of devcontainers spec/CLI. However the vscode team who owns the vscode remote containers might be able to evaluate this and provide directions.
Could you please open an issue at https://github.com/microsoft/vscode-remote-release/issues so that the vscode team can take a look at it?
